runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -245,6 +245,7 @@
               -->
               <excludedGroups>
                 org.apache.beam.sdk.testing.LargeKeys$Above10MB,
+                org.apache.beam.sdk.testing.UsesAttemptedMetrics,
```

Comment:
This doesn't seem to be exclusively Dataflow Batch. How does this affect Streaming tests?